src/library_webgpu.js

```diff
@@ -1317,9 +1312,10 @@ var LibraryWebGPU = {
 
     function makeVertexBuffer(vbPtr) {
       if (!vbPtr) return undefined;
-      return {
+      var stepModeValue = WebGPU.VertexStepMode[{{{ gpu.makeGetU32('vbPtr', C_STRUCTS.WGPUVertexBufferLayout.stepMode) }}}];
+      return stepModeValue === undefined ? undefined :{
```

Comment:
nit: I think returning `undefined` works but the IDL actually has this nullable, so should return `null` to be more precise.